integration-test/src/main/resources/application.yml

```diff
@@ -69,6 +69,8 @@ integrationtest:
   upgrade:
     currentRuntimeVersion: 7.2.0
     targetRuntimeVersion: 7.2.6
+  privateendpoint:
+    enabled: false
```

Comment:
If `privateEndpoint` won't contain any further parameters, you can set this directly:
```
privateEndpoint: false
```

Reply:
I changed it to privateEndpointEnabled, would that be ok?